### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by Pinkish-Warrior**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ import Bars from "./components/Bars";
 import UserInputForm from "./components/UserInputForm";
 import MapComponent from "./components/MapComponent";
 import Header from "./components/Header"
+import Footer from "./components/Footer/Footer";
+import SocialFollow from "./components/Footer/SocialFollow";
```
Comment (s-palmer) [inline src/App.js]: I don't think you need to import socialfollow here

File: src/components/Footer/SocialFollow.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import "./Footer.css";
+
+
+
+const SocialFollow = () => {
+  return (
+    <div className='social-container' >
+      <h3> Swep-III @makers</h3>
+      <a 
+      href='https://github.com/makersacademy' className="social"
```
Comment (s-palmer) [inline src/components/Footer/SocialFollow.js:17]: maybe we should link to our github repo here for the project?

File: src/components/Footer/SocialFollow.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  faGoogle
+} from '@fortawesome/free-brands-svg-icons';
+import "./Footer.css";
+
+
+
+const SocialFollow = () => {
+  return (
+    <div className='social-container' >
+      <h3> Swep-III @makers</h3>
```
Comment (s-palmer) [inline src/components/Footer/SocialFollow.js:15]: Here we could add a link to makers instead perhaps?

File: src/components/Footer/SocialFollow.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      >
+      <FontAwesomeIcon icon={faReact} size='2x' />
+      </a>
+      <a 
+      href='https://nodejs.org/en/' className="social"
+      >
+      <FontAwesomeIcon icon={faNode} size='2x' />
+      </a>
+      <a 
+      href='https://console.cloud.google.com/apis/dashboard?project=resonant-augury-333910' className="social"
```
Comment (s-palmer) [inline src/components/Footer/SocialFollow.js]: Maybe just link to the general google cloud page instead of your personal dashboard?

**Comment by Pinkish-Warrior** [comment]
Sorry it meant to be a link to the google api  main page as one of the tools we use, however as was signed in and I believe it linked it. Thanks 
Have you fixed it?! Or should I?! 

Tania Rosa Santana
She/her

**Comment by s-palmer** [comment]
> Sorry it meant to be a lint to the google api main page as one of the tools we use, however as was signed in and I believe it linked it. Thanks Have you fixed it?! Or should I?! Tania Rosa Santana She/her
> […](#)
> On 7 Dec 2021, at 15:14, Sergei Palmer ***@***.***> wrote: ﻿ @s-palmer commented on this pull request. In src/components/Footer/SocialFollow.js: > + href='https://github.com/makersacademy' className="social" + > + <FontAwesomeIcon icon={faGithub} size='2x' /> + </a> + <a + href='https://reactjs.org/' className="social" + > + <FontAwesomeIcon icon={faReact} size='2x' /> + </a> + <a + href='https://nodejs.org/en/' className="social" + > + <FontAwesomeIcon icon={faNode} size='2x' /> + </a> + <a + href='https://console.cloud.google.com/apis/dashboard?project=resonant-augury-333910' className="social" Maybe just link to the general google cloud page instead of your personal dashboard? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

I'll let you fix it!

**Comment by s-palmer** [comment]
I think quite a lot may have changed since this branch was opened. I think you'll need to pull the main updates into this branch first as it has some conflicts to be merged

File: src/components/Footer/SocialFollow.js (hunk 2 above)
Comment (Pinkish-Warrior) [reply src/components/Footer/SocialFollow.js:17]: Sorry it meant to be a link to the google api  main page as one of the tools we use, however as was signed in... I believe it linked it that. Thanks

File: src/components/Footer/SocialFollow.js (hunk 3 above)
Comment (Pinkish-Warrior) [reply src/components/Footer/SocialFollow.js:15]: great idea!!! will do

**Pull request closed without merging**